install/kubernetes/values.yaml

```diff
@@ -52,7 +52,7 @@ tetragon:
   enabled: true
   image:
     override: ~
-    repository: quay.io/cilium/tetragon
+    repository: quay.io/cilium/tetragon-ci
```

Comment:
This makes sense if the values in the `main` branch are pointing to latest, since we have the release gating in place to ensure that `quay.io/cilium/tetragon` requires maintainer signoff to push new images there (ie only for full official releases). FWIW in Cilium, for release branches and tags we then go and rewrite the `tag` below to the relevant tag when we release the helm charts. At the same time we would do the equivalent of ensuring that this repository gets switched back to the main repo like `quay.io/cilium/tetragon`.

Probably just need to integrate this into whatever release scripting we set up.